src/ajax.ts

```diff
@@ -37,7 +52,31 @@ function getConnector() {
  * @returns {Promise<any>}
  */
 export function send(url: string, data : any = {}, method = 'get', expectedDataType = 'json'): Promise<any> {
-  return getConnector().then((c) => c.send(url, data, method, expectedDataType));
+  if (method === 'get' || method === 'head') {
+    data = encodeParams(data); //encode in url
+    if (data) {
+      url += (/\?/.test(url) ? '&' : '?') + data;
+    }
+    data = null;
```

Comment:
Reassigning variables can make things a bit harder to maintain... Here it looks like the point of assigning it null is to keep it from going into the "if" below... Maybe if would be more clear to make that into an else, so it's more obvious that there are two mutually exclusive possibilities?